cola/widgets/diff.py

```diff
@@ -689,6 +691,12 @@ def __init__(self, context, options, parent):
 
         # "Diff Options" tool menu
         self.options = options
+
+        # Enabling Word Wrapping by default
+        self.setWordWrapMode(QtGui.QTextOption.WordWrap)
+        self.setLineWrapMode(QtWidgets.QPlainTextEdit.WidgetWidth)
+        self.options.enable_word_wrap.setChecked(True)
+
```

Comment:
If we add the standalone function then this section should also condense down into a single call.